orchestra/static/orchestra/todos/todo-qa.directive.es6.js

```diff
@@ -14,8 +14,38 @@ export default function qa () {
     controller: function (todoApi, todoQaApi, $scope) {
       var todoQa = this
       todoQa.todos = []
+      todoQa.project = null
       todoQa.ready = false
 
+      todoQa.copyToClipboard = str => {
+        const el = document.createElement('textarea')
+        el.value = str
+        document.body.appendChild(el)
+        el.select()
+        document.execCommand('copy')
+        document.body.removeChild(el)
+      }
+
+      todoQa.commentSummary = (todos, summary) => {
+        summary = summary || ''
+        todos.forEach((todo) => {
+          summary = todoQa.commentSummary(todo.items, summary)
+          if (todo.qa && !todo.qa.approved && todo.qa.comment) {
+            summary = `*Todo*: ${todo.description}\n*Comment*: ${todo.qa.comment}\n\n${summary}`
+          }
+        })
+        return summary
+      }
+
+      todoQa.qaSummary = () => {
+        const commentSummary = todoQa.commentSummary(todoQa.todos)
+        if (commentSummary.length) {
+          return `*Feedback on ${todoQa.project.short_description}*\n\n${commentSummary}`
+        } else {
+          return `*No feedback on ${todoQa.project.short_description}*`
```

Comment:
Not about this PR. Interestingly, we give feedback on their error but not on what they do well (e.g. Great job picking out the hero images for the business!).

Reply:
I thought about it. I used only disapproved todos to keep it consistent with the logic for showing feedback on their todos.

Reply:
Now showing comments from approved todos as well.